client/src/components/Banner/Banner.js

```diff
@@ -2,7 +2,13 @@ import React from "react";
 import "./Banner.css";
 
 const Banner = props => {
-  return <div className="Banner" style={{
+  let classList = '';
+  if (props.bannerType === 'BannerLarge' ) {
+    classList += ` Banner--large`
+  } else {
+    classList += ` Banner`
+  }
+  return <div className={classList} style={{
```

Comment:
`Banner--large` is a modifier, so it should be added in addition to `Banner`. I.e. `Banner Banner--large`

Reply:
This is where CSS becomes composable and lets you write dry CSS. I.e. all the properties you have repeated between these two classes can be removed no!

Reply:
implemented recommendation above and removed repetitive CSS.